compiler/qsc_parse/src/item/tests.rs

```diff
@@ -41,6 +42,18 @@ fn adjoint_invert() {
     );
 }
 
+// unit tests for file_name_to_namespace_name
+#[test]
+fn test_file_name_to_namespace_name() {
+    let raw = "foo/bar.qs";
+    let error_span = Span::default();
+    let namespace =
+        source_name_to_namespace_name(raw, error_span).expect("test should not fail here");
+    assert_eq!(namespace.0.len(), 2);
+    assert_eq!(&*namespace.0[0].name, "foo");
+    assert_eq!(&*namespace.0[1].name, "bar");
```

Comment:
Just cause this is already the pattern in this file: this test would read clearer if you used an `expect![]` to validate the object returned. 

Also nitpicking: drop the `test_` prefix from the function name since none of the other tests in this file use it.